configuration/white_labels/co.py

```diff
@@ -2109,37 +2109,57 @@ def get_white_label(self) -> WhiteLabel:
         },
         "healthCare": {
             "benefits": {
+                "connect_for_health_premium_tax_credit": {
```

Comment:
I want to start prefixing things with the state code.
```suggestion
                "co_cfhc": {
```